courses/free-courses-te.md

```diff
@@ -1,23 +1,87 @@
 ### Index
 
+* [Adobe Illustrator](#adobe-illustrator)
+* [Angular](#angular)
+* [ASP](#asp)
+* [Bash Shell Scripting](#bash-shell-scripting)
+* [Blender](#blender)
+* [Bootstrap](#bootstrap)
 * [C](#c)
+* [C#](#csharp)
 * [C++](#cpp)
+* [Flutter](#flutter)
+* [HTML](#html)
 * [Java](#java)
 * [Javascript](#javascript)
+* [JSON](#json)
+* [JQuery](#jquery)
+* [Kotlin](#kotlin)
+* [MySQL](#mysql)
+* [PowerShell](#powershell)
 * [Python](#python)
 * [R](#r)
+* [ReactJS](#reactjs)
+* [SQL](#sql)
+* [SVG](#svg)
+* [Unity](#unity)
+* [Unreal Engine](#unreal-engine)
+
+
+### Adobe Illustrator
+
+* [illustrator tutorials in telugu](https://www.youtube.com/playlist?list=PLp0XgOsoW2Xb0S3Od1YgqK5FacIldY7-t) - SEO Skills (YouTube)
+
+
+### Angular
+
+* [Angular 10 in telugu](https://www.youtube.com/playlist?list=PLKIBeLKGEbp4cZdaLtOAVw8BH5VHXC3fT) - RSK Helpline (YouTube)
+
+
+### ASP
+
+* [ASP .Net full course in Telugu](https://www.youtube.com/playlist?list=PL2UfVCyiPs-06uZQAEvbdUH7vUHvlGuj5) - Edusoft Learning Systems - Learning Simplified (YouTube)
+
+
+### Bash Shell Scripting
+
+* [Shell Scripting Tutorials in Telugu](https://www.youtube.com/playlist?list=PLZn2KEoSSSgGVHkjSGjZ0HU08FskbACxR) - 7 Hills (YouTube)
+
+
+### Blender
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Reply:
Yes, added multiple resources for each category, and added author names.